README.md

```diff
@@ -22,12 +22,17 @@ Develop branch status
 
 #quickstart instructions
 
+0. Clone data_api repo
+
+        git clone https://github.com/kbase/data_api/
+
 1. install virtualenv and python development libraries using your local installer::
 
         apt-get install python-dev python-virtualenv
 
 2. create a virtualenv environment to install the source to::
-
+        
+        cd data_api
         virtualenv venv
 
 3. activate the virtualenv::
```

Comment:
I don't see it in the diff, but it's just because it got cut off and I wasn't paying enough attention.  I think it's fine as-is.